neuro_san/internals/graph/activations/intra_agent_message_utils.py

```diff
@@ -44,13 +61,13 @@ def generate_response(the_messages: List[Any]) -> str:
                 if previous_role == "assistant":
                     new_message = {
                         "role": previous_role,
-                        "content": IntraAgentMessageUtils.get_content(previous_message)
+                        "content": IntraAgentMessageUtils._get_content(previous_message)
```

Comment:
Make all the helper methods in this class private.  The only entrypoint is this generate_response() method.